stock_scanner_web/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Stock Scanner Web
+=================
+
+This module extends the functionality of stock scanner play scenarios on web
+interface. Some features of this module are:
+
+*  Responsive layout, properly displayed on mobiles.
+*  Link called "Web Stock Scanner" in the inventory menu to access the website.
+*  Added modifications to restrict hardware usage to specific users.
+*  Initial View to allow user to select among various hardware devices.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Create a Scanner Hardware and add to the user. You can now do this
+directly from the user form.
+* Also, do not forget to enable "Sentinel: technical users" option in the user
+form, as it is required by the basic stock_scanner module.
+* Go to "Inventory > Configuration > Scanner Configuration > Scanner
```

Comment:
Yes, because the login is checked when the user enters the odoo database, so we thought is not needed to ask for credentials again in the scanner web as the access it's done through the inventory menu. The web will show to the user his/her hardware allowed to use directly and check for permissions if he or she try to access another hardware via typing other device's address.
In order to make possible to use web and sentinel at the same time We think the login/logout option should be moved to the hardware configuration. This way you can specify no login/logout for hardware using web and enable it for the sentinel.

cc @sylvain-garancher